content/en/core/overview/architecture.md

```diff
@@ -8,31 +8,59 @@ description: >
 
 ## Server side
 
-### CouchDB
+### CHT Core Framework
 
-A free and open source NoSQL database we use to store all our data, configuration, and even the application code. CouchDB is really good at replication which is the process of sending the data to another database and back again. See more at the [CouchDB](http://couchdb.apache.org) site.
+The cht-core product is the primary component of the CHT. The server comes with authentication, role based authorization, data security, and a range of protected data access endpoints. Read more detail in [cht-core GitHub repository](https://github.com/medic/cht-core).
 
-### api
+#### api
 
-A NodeJS service which runs on the server as a wrapper around CouchDB. It provides security and APIs for browsers and integrations. It also includes a custom implementation of filtered replication to allow it to support more concurrent users. See more at the [CHT Core API repo](https://github.com/medic/cht-core/tree/master/api) on Github.
+A NodeJS service which runs on the server and provides security and APIs for browsers and integrations. It also includes a custom implementation of filtered replication to allow it to support more concurrent users. See more at the [CHT Core API repo](https://github.com/medic/cht-core/tree/master/api) on Github.
 
-### sentinel
+#### sentinel
 
 Another NodeJS service running on the server, sentinel performs actions called transitions every time a document in CouchDB is added or modified. Some examples are validations, generating scheduled messages, automatic responses, creating patients, and sending alerts. See more at the [CHT Core Sentinel repo](https://github.com/medic/cht-core/tree/master/sentinel) on Github. 
 
-### PostgreSQL
+#### CouchDB
 
-A free and open source SQL database that we use for analytics queries for display in tools like klipfolio. We created a library called [couch2pg](https://github.com/medic/couch2pg) to replicate data from CouchDB into PostgreSQL. See more at the [PostgreSQL](https://www.postgresql.org) site.
+A free and open source NoSQL database used to store all data and configuration. This can be multiple instances clustered together for additional scalability. CouchDB is really good at replication which is the process of sending the data to another database and back again, which makes it ideal for replicating data to the phone for offline access. See more at the [CouchDB](http://couchdb.apache.org) site.
+
+#### nginx
+
+Provides SSL termination and routes requests to API.
+
+#### haproxy
+
+Provides audit logging for any request that makes it to CouchDB so any data access or modification can be validated at a later date.
+
+### CHT Sync
+
+The cht-sync product is a suite of tools for extracting and normalizing data from the Core Framework's CouchDB and rendering the data in analytics dashboards to measure the performance of the program. Read more detail in [cht-sync GitHub repository](https://github.com/medic/cht-sync).
```

Comment:
I find it awkward to start with the repo name "cht-sync", suggesting to avoid using it right away in the sentence.

```suggestion
A suite of tools for extracting and normalizing data from the Core Framework's CouchDB and rendering the data in analytics dashboards to visualize key data for a CHT deployment. Read more detail in [`cht-sync` GitHub repository](https://github.com/medic/cht-sync).
```